requirements.txt

```diff
@@ -5,3 +5,4 @@ delta-utils
 gym
```

Comment:
This file is missing `torch`

Reply:
Or is that deliberate to get the right version of `torch`? Can't we just specify that here?

Reply:
Yeah torch purposefully not in here because a pip install -r requirements.txt will grab the latest non-cpu version. I've tried previously to specify our special version but i'm not sure it's possible :(

Reply:
This seems to work locally :smile: 
```
# Torch
--find-links https://download.pytorch.org/whl/torch_stable.html
torch==1.8.0+cpu
```

Reply:
:octocat: